mopidy/local/__init__.py

```diff
@@ -24,7 +24,7 @@ def get_config_schema(self):
         schema['library'] = config.String()
         schema['media_dir'] = config.Path()
         schema['data_dir'] = config.Path()
-        schema['playlists_dir'] = config.Path()
+        schema['playlists_dir'] = config.Deprecated()
```

Comment:
Doesn't look like we currently do anything with `config.Deprecated()` except not throwing a fit over finding an unknown config value in the file. Feels out of scope for this PR, but surely something we should consider adding.